src/views/product-downloads-view/components/downloads-section/index.tsx

```diff
@@ -0,0 +1,259 @@
+import { ReactElement, useMemo } from 'react'
+import classNames from 'classnames'
+import { IconExternalLink16 } from '@hashicorp/flight-icons/svg-react/external-link-16'
+import CodeBlock from '@hashicorp/react-code-block'
+import { PackageManager } from 'views/product-downloads-view/types'
+import { prettyOs } from 'views/product-downloads-view/helpers'
+import Card from 'components/card'
+import DownloadStandaloneLink from 'components/download-standalone-link'
+import Heading, { HeadingProps } from 'components/heading'
+import InlineLink from 'components/inline-link'
+import Tabs, { Tab } from 'components/tabs'
+import Text from 'components/text'
+import StandaloneLink from 'components/standalone-link'
+import { DownloadsSectionProps } from './types'
+import {
+  generateCodePropFromCommands,
+  groupDownloadsByOS,
+  groupPackageManagersByOS,
+} from './helpers'
+import s from './downloads-section.module.css'
+
+const SHARED_HEADING_LEVEL_3_PROPS = {
+  className: s.subHeading,
+  level: 3 as HeadingProps['level'],
+  size: 200 as HeadingProps['size'],
+  weight: 'semibold' as HeadingProps['weight'],
+}
```

Comment:
I wonder if this would work 💭 
```suggestion
const SHARED_HEADING_LEVEL_3_PROPS: Partial<HeadingProps> = {
  className: s.subHeading,
  level: 3,
  size: 200,
  weight: 'semibold',
}
```

Reply:
I've not seen this before! I'll give it a try and see what happens

Reply:
No luck on that one. Still getting the error I was resolving with all the type casting.

![image](https://user-images.githubusercontent.com/43934258/155754676-01407e47-1c77-4957-b93b-0afea012de01.png)